core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -1966,6 +1967,20 @@ public static TimeZone timeZone(DataContext root) {
     return (TimeZone) DataContext.Variable.TIME_ZONE.get(root);
   }
 
+  /** SQL {@code USER} function. */
+  @Deterministic
+  public static String user(DataContext root) {
+    return java.util.Optional.ofNullable((String) DataContext.Variable.USER.get(root))
```

Comment:
Pls remove `java.util` package. `java.util.Optional` is already imported.

Reply:
done.